rollup.config.ts

```diff
@@ -135,10 +138,13 @@ export default async function (
 		],
 		plugins: [
 			replaceBrowserModules(),
+			replace(wasmReplacement),
+			replace(getEnvironmentReplacement(true)),
 			alias(moduleAliases),
 			nodeResolve({ browser: true }),
 			json(),
 			commonjs(),
+			wasm(),
```

Comment:
I don't set the `targetEnv` option to `browser` for running browser test successfully. But it is generating compatible code for different environments.
![image](https://github.com/rollup/rollup/assets/33021497/bc27d2d4-9e04-46a7-8748-cf420c2e72df)

Reply:
But doesn't that mean the browser build also contains Node code that we do not need? Of course it makes sense to also have a Node Wasm build, but then I would say should be a proper build with CLI published under a separate name. The browser build is really hard to use as it of course needs plugins for everything.

Reply:
I set the `targetEnv` option to `browser`. I'm now thinking that how to fix browser tests, maybe we need a headless browser such as playwright to run browser tests?

Reply:
Yes, I think so too.

Reply:
BTW CI tests should be green now (everything that is not working is commented out), there is a TypeScript issue in your code and maybe you need to add some comments to silence the linter.

Reply:
I tried using `puppeteer` to run browser tests, but it's not easy to migrate current browser tests due to runtime context. So I polyfilled some browser global variables like `fetch` and `WebAssembly.compileStreaming`, maybe this is also an appropriate solution?

Reply:
Sure, I think this is fine for now.